test-apps/presentation-test-app/src/frontend/components/properties-widget/PropertiesWidget.tsx

```diff
@@ -170,6 +177,8 @@ function PropertyGrid(props: PropertyGridProps) {
 
   return <>
     <VirtualizedPropertyGridWithDataProvider
+      width={width}
+      height={height}
       dataProvider={filteringDataProvider}
       isPropertyHoverEnabled={true}
       onPropertyContextMenu={onPropertyContextMenu}
```

Comment:
Property Grid in presentation-test-app no longer switches to vertical mode when it crosses the width threshold.

Reply:
fixed

Reply:
Thanks